scheduler/db/scheduler.changelog-master.yaml

```diff
@@ -160,6 +160,11 @@ databaseChangeLog:
                   constraints:
                     primaryKey: true
               - column:
+                  name: timestamp

```

Comment:
Should be a timestamp type (see api)
The default value for this column should be NOW().
We don't need to insert or touch it.